codec-http/src/main/java/io/netty/handler/codec/http/RFC6265ClientCookieEncoder.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2012 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.http;
+
+import static io.netty.handler.codec.http.CookieEncoderUtil.*;
+
+/**
+ * A RFC6265 compliant cookie encoder to be used client side, so only name=value pairs are sent.
+ *
+ * User-Agents are not supposed to interpret cookies, so, if present, Cookie.rawValue will be used.
+ * Otherwise, Cookie.value will be used unquoted.
+ *
+ * Note that multiple cookies are supposed to be sent at once in a single "Cookie" header.
+ *
+ * <pre>
+ * // Example
+ * {@link HttpRequest} req = ...;
+ * res.setHeader("Cookie", {@link RFC6265ClientCookieEncoder}.encode("JSESSIONID", "1234"));
+ * </pre>
+ *
+ * @see RFC6265ClientCookieDecoder
+ */
+public final class RFC6265ClientCookieEncoder {
+
+    /**
+     * Encodes the specified cookie into an HTTP header value.
+     */
+    public static String encode(String name, String value) {
+        return encode(new DefaultCookie(name, value));
+    }
+
+    public static String encode(Cookie cookie) {
+        if (cookie == null) {
+            throw new NullPointerException("cookie");
+        }
+
+        StringBuilder buf = stringBuilder();
+        encode(buf, cookie);
+        return stripTrailingSeparator(buf);
+    }
+
+    public static String encode(Cookie... cookies) {
+        if (cookies == null) {
+            throw new NullPointerException("cookies");
+        }
+
```

Comment:
I'm fine as long as this behavior is documented.  It seems like `encode(Cookie)` and `encode(Cookie...)` do not have Javadoc.  Could you add some?